charts/metallb/templates/speaker.yaml

```diff
@@ -482,19 +495,14 @@ spec:
           - --logtostderr
           - --secure-listen-address=:{{ .Values.speaker.frr.secureMetricsPort }}
           - --tls-cipher-suites=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_RSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
-          - --upstream=http://$(METALLB_HOST):{{ .Values.speaker.frr.metricsPort }}/
+          - --upstream=http://127.0.0.1:{{ .Values.speaker.frr.metricsPort }}/
```

Comment:
It seems you also missing it here :)

Reply:
Fixed, good catch! Let's hope it won't implode again :P